lib/wifi_user/gateway/gov_notify_sms.rb

```diff
@@ -0,0 +1,31 @@
+require 'notifications/client'
+
+class WifiUser::Gateway::GovNotifySMS
+  def initialize(api_key)
+    @client = Notifications::Client.new(api_key)
+  end
+
+  def execute(phone_number, template_id, template_parameters: {})
```

Comment:
Decided to stick with template_id in the execute? 

Agreed that it felt weird, but don't feel strongly about moving it right this minute.

Reply:
yeah, it'll cause a lot of mess as currently most template IDs are fetched from the template finder.

It might be that at some point, the template finder becomes something that dishes out gateways, but that's for the future when this project gets refactored